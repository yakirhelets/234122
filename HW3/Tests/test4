getApartments sourceOne "Kendrick Lamar" 3
