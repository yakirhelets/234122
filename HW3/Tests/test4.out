13 5000 050-0000123
12 4000 050-0000123
15 2000 050-0000123
